### PR TITLE
docs(changelog): for v13.0.0 [skip ci]

### PR DESCRIPTION
**Pull request by tdeekens**
We will first try out the canary release. Eventually this is the release notes when hitting latest.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,20 @@
+## [13.0.0](https://github.com/commercetools/merchant-center-application-kit/compare/v12.2.2...v13.0.0) (2019-04-04)
+
+This release introduces a **breaking change** which however will not entail any **migration steps** as it only affects internal packages. However, we want to follow strict semver and therefore we should mark it as a major version bump.
+
+In short:
+
+1. A new package called `mc-dev-authentication` now contains routes and views for login and logout shared by both the `webpack-dev-server` and `mc-http-server`
+2. All routes and components around login and logout have been removed from the `@commercetools-frontend/application-shell` and have been migrated into an internal Merchant Center application
+3. Redirects when signing up (still disabled) will be performed allowing users to create their first organization and project
```
Comment (emmenko) [inline CHANGELOG.md]:
```suggestion
3. Redirects upon creating an account are not pointing to the account application, where the user can create the first organization + project
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,20 @@
...
+
+There is far more internal documentation available internally to any commercetools developer. To not hestitate to reach out.
```
Comment (emmenko) [inline CHANGELOG.md]:
```suggestion
For commercetools developers there is more in-depth documentation in our internal repositories. To not hestitate to reach out for further questions.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,20 @@
...
+
+#### 💥 Type: Breaking Change
+
+- `application-shell-connectors`, `application-shell`, `constants`, `i18n`, `mc-dev-authentication`, `mc-http-server`, `mc-scripts`
+  - [#444](https://github.com/commercetools/merchant-center-application-kit/pull/444) Remove login/logout components and redirect to urls ([@tdeekens](https://github.com/tdeekens))
```
Comment (emmenko) [inline CHANGELOG.md]:
```suggestion
  - [#444](https://github.com/commercetools/merchant-center-application-kit/pull/444) Remove login/logout components ([@tdeekens](https://github.com/tdeekens))
```

**Comment by emmenko** [comment]
PS: I added the release note for the new locales

**Comment by tdeekens** [comment]
Sure, thing thanks.

**Pull request merged**